tests/minicrater/resource-exhaustion/results.expected.json

```diff
@@ -23,15 +23,15 @@
         "Local": "memory-hungry"
       },
       "name": "memory-hungry (local)",
-      "res": "spurious-fixed",
+      "res": "test-pass",
       "runs": [
         {
           "log": "stable/local/memory-hungry",
-          "res": "build-fail:oom"
+          "res": "test-pass"
         },
         {
           "log": "beta/local/memory-hungry",
-          "res": "test-fail:oom"
+          "res": "test-pass"
```

Comment:
This should not have been blessed.

Reply:
How can I make sure my changes are right if they're different on CI and locally?

Reply:
I ended up only running `minicrater::doc` which seems to work well enough. But I'd have the same question if I needed to modify `resource_exhaustion`.